components/automate-chef-io/content/docs/ldap.md

```diff
@@ -1,82 +1,308 @@
+Chef Automate's default configuration for Microsoft AD is specific to LDAP.
+To configure Microsoft AD using SAML, see the [SAML documentation]({{< relref "saml.md" >}}).
 
-### Authorization with LDAP
+## Changing Chef Automate Configuration
 
-Chef Automate supports using your LDAP server to authenticate users and define their permissions based on [LDAP group memberships]({{< ref "authorization-overview.md" >}}).
+If you need to change your configured external identity provider settings, replace your existing configuration by following these steps:
 
-#### Connect
+1. Run `chef-automate config show config.toml`.
+2. Edit `config.toml` to replace the `dex.sys.connectors` section with the configuration values for your new identity provider.
+3. Run `chef-automate config set config.toml` to set your updated configuration.
+
+### Minimal MSAD Configuration
+
+base_user_search_dn
+: "your base user search DN"
+
+base_group_search_dn
+: "your base group search DN"
+
+bind_dn
+: "your bind_dn"
+
+bind_password
+: "your bind_password"
+
+ca_contents
+: Your certificate authority (CA) certificate contents. You can provide multiple PEM-encoded CA certs. Optional.
 
-Chef Automate first needs to establish a TCP connection to your LDAP service,
-secured by TLS.  It will connect to the host configured in your TOML config, for
-example
+  ```toml
+  # Example ca_contents setting:
+  ca_contents = """-----BEGIN CERTIFICATE-----
+  MIICsDCCAhmgAwIBAgIJAJxMopMJbhPkMA0GCSqGSIb
+  ...
+  X0uRzUPlpttd5tYFs43nkqxJT6s=
+  -----END CERTIFICATE-----"""
+  ```
+
+host
+: The domain name of your directory server, for example `"ldap.corp.com"`. Default port: `636`. Override the port by appending it to the host setting, `"ldap.corp.com:10636"`
+
+#### Minimal MSAD config.toml
 
 ```toml
-host = "ldap.corp.com"
+[dex.v1.sys.connectors.msad_ldap]
+host = "<your host>"
+bind_dn = "<your bind_dn>"
+bind_password = "<your bind_password>"
+base_user_search_dn = "<your base user search DN>"
+base_group_search_dn = "<your base group search DN>"
+ca_contents = "<your ca contents>" # optional, but recommended
 ```
 
-Automate uses port `636` by default.  To override the port, append it to the
-host setting, e.g.
+### Full MSAD Configuration
+
+The MSAD configuration is an LDAP configuration with more provided default values that are commonly a good fit for Active Directory. Override any single default value by uncommenting it in the configuration and setting its value:
+
+email_attr
+: "mail"
+
+filter_groups_by_user_attr
+: "member"
+
+filter_groups_by_user_value
+: "DN"
+
+group_query_filter
+: "(objectClass=group)"
+
+group_display_name_attr
+: "displayName"
+
+insecure_no_ssl
+: false
+{{% warning %}}
+Connecting to an LDAP service without TLS is not recommended.
+{{% /warning %}}
+
+user_display_name_attr
+: "displayName"
+
+user_id_attr
+: "sAMAccountName"
+
+user_query_filter
+: "(objectClass=person)"
+
+username_attr
+: "sAMAccountName"
```

Comment:
Similar to above, I do not find where L108-L139 come from in the original and again, they seem to be already covered by L145-L162 so consider deleting L108-L139.